zerver/views/user_settings.py

```diff
@@ -41,7 +41,8 @@ def confirm_email_change(request: HttpRequest, confirmation_key: str) -> HttpRes
         raise JsonableError(_("Email address changes are disabled in this organization."))
     do_change_user_email(user_profile, new_email)
 
-    context = {'realm': user_profile.realm, 'new_email': new_email}
+    context = {'realm': user_profile.realm.name, 'new_email': new_email}
+
```

Comment:
extra newline

Reply:
Sorry, I just didn't understand how templates work:smiley:
rolled this change back now